packages/yonode-templates/JS-MySQL-Prisma-NoAuth-Template/src/controllers/helloWorldController.js

```diff
@@ -1,11 +1,10 @@
-export const hellWorld = async (res, req)=>{
-    try{
-
-     res.send("Hello world!")
-
-    }catch(err){
-      res.status(500).json({
-        message: "Internal Server Error"
-      })
-    }
+export const hellWorld = async (req, res)=>{
```

Comment:
fix hellworld to helloworld. now its hell instead of hello

Reply:
didn't see the typo, I fixed it in all of the templates